packages/ui/src/modal/acceptBid/AcceptBidModal.tsx

```diff
@@ -0,0 +1,376 @@
+                    marginBottom: 16,
+                    marginTop: 16,
+                    marginRight: 16,
+                    marginLeft: 16,
+                  }}
+                  color="primary"
+                  onClick={acceptBid}
+                >
+                  Accept
+                </Button>
+              </Flex>
+            )}
+
+            {(acceptBidStep === AcceptBidStep.Confirming ||
+              acceptBidStep === AcceptBidStep.Finalizing) &&
+              token && (
+                <Flex direction="column">
+                  <TokenLineItem
+                    tokenDetails={token}
+                    collection={collection}
+                    usdConversion={ethUsdPrice || 0}
+                    price={price}
+                    warning={warning}
+                    currency={currency}
+                    expires={expires}
+                  />
+                  <Progress
+                    acceptBidStep={acceptBidStep}
+                    etherscanBaseUrl={`${etherscanBaseUrl}/tx/${txHash}`}
+                    marketplace={marketplace}
+                    tokenImage={tokenImage}
+                  />
+                  <Button disabled={true} css={{ m: '$4' }}>
+                    <Loader />
+                    {acceptBidStep === AcceptBidStep.Confirming
+                      ? 'Waiting for approval...'
+                      : 'Waiting for transaction to be validated'}
+                  </Button>
+                </Flex>
+              )}
+
+            {acceptBidStep === AcceptBidStep.Complete && token && (
+              <Flex direction="column">
+                <Flex
+                  css={{
+                    p: '$4',
+                    py: '$5',
+                    flexDirection: 'column',
+                    alignItems: 'center',
+                    textAlign: 'center',
+                  }}
+                >
+                  {' '}
+                  <Box
+                    css={{
+                      color: '$successAccent',
+                      mb: 24,
+                    }}
+                  >
+                    <FontAwesomeIcon icon={faCheckCircle} fontSize={32} />
+                  </Box>
+                  <Text style="h5" css={{ mb: 8 }}>
+                    Bid accepted!
+                  </Text>
+                  <Flex
+                    css={{ mb: 24, maxWidth: '100%' }}
+                    align="center"
+                    justify="center"
+                  >
+                    <Text
+                      style="subtitle2"
+                      css={{ maxWidth: '100%' }}
+                      ellipsify
+                    >
+                      You’ve sold{' '}
+                      <Anchor
+                        color="primary"
+                        weight="medium"
+                        css={{ fontSize: 12 }}
+                        href={`https://dev.reservoir.market/${token.token?.contract}/${token?.token?.tokenId}`}
```

Comment:
These hardcoded urls should use the redirect api with the marketplace source. You can use the client hook to get the client and access the source.